controllers/managedocs_controller.go

```diff
@@ -172,6 +173,7 @@ type ManagedOCSReconciler struct {
 // +kubebuilder:rbac:groups="network.openshift.io",namespace=system,resources=egressnetworkpolicies,verbs=create;get;list;watch;update
 // +kubebuilder:rbac:groups="coordination.k8s.io",namespace=system,resources=leases,verbs=create;get;list;watch;update
 // +kubebuilder:rbac:groups="odf.openshift.io",namespace=system,resources=storagesystems,verbs=list;watch;delete
+// +kubebuilder:rbac:groups="config.openshift.io",resources=clusterversions,verbs=get
```

Comment:
```suggestion
// +kubebuilder:rbac:groups="config.openshift.io",resources=clusterversions,verbs=get;watch
```

Reply:
Tried this as well, didn't work

Reply:
@akshayashu 
- generally, for Read on api in CRUD (create, read, update, delete) ops you need `verbs=get;watch;list`
- rationale being, the sdk during setting up shared caches lists the resources and then performs ops on cache
- this should be the reason why you are seeing `forbidden` error mentioned in other comment

Reply:
@leelavg Adding list didn't work too.

Reply:
It worked with `verbs=get;watch;list`

Reply:
@akshayashu Can you resolve this conversation?